documentation/modules/configuring/con-config-large-messages.adoc

```diff
@@ -0,0 +1,157 @@
+// Module included in the following assemblies:
+//
+// assembly-config-high-volumes.adoc
+
+[id='con-config-large-messages-{context}']
+= Handling large message sizes
+
+Kafka's default batch size for messages is 1MB, which is optimal for maximum throughput in most use cases.
+Kafka can accommodate larger batches at a reduced throughput, assuming adequate disk capacity.
+
+Large message sizes are handled in four ways:
```

Comment:
The `are` here suggests something does that for the user. But the user actually need to implement some of them. So you something like `can be`
 better?

```suggestion
Large message sizes can be handled in four ways:
```